src/providers/inventory-data.ts

```diff
@@ -17,28 +17,23 @@ export class InventoryData {
   }
 
   addItem(item: Object) {
-    return this.authHttp.put(ApiUrl + Links.item, item)
-      .map(this.extractData);
+    return this.putEndpoint(Links.item, item);
   }
 
   editItem(item: Object, tag: string) {
-    return this.authHttp.put(ApiUrl + Links.item + '/' + tag, item)
-      .map(this.extractData);
+    return this.putEndpoint(Links.item + '/' + tag, item);
```

Comment:
I don't really have a preference. Template literals felt weird when I first started to use them, but I got used to it. I also couldn't find any guidelines from Angular about which one to use. However, I think ES6 recommends using that syntax and there seems to be a few advantages to it, so I will create an issue to change the string concatenation throughout the app to template literals.